Rakefile

```diff
@@ -435,12 +450,21 @@ which are simply available in the repository.
     run_build_cmd(cmd, SPHINX_BUILD_OUTFILE)
   end
 
-  desc <<-EOF
+  desc <<-EOF.gsub(/^ {4}/,'')
     Run a local web server to view HTML docs on http://localhost:port'
 
     ARGS:
       * port => The port the local webserver will listen on (default: 5000)
 
+    ENVIRONMENT VARIABLES:
+      * SIMP_DOC_extra_sphinx_args => additional arguments to inject into
+                                      invocation of `sphinx-build` or `sphinx-autobuild`
+      * SIMP_FAST_DOCS             => (used inside conf.py) defaults to 'true'
+                                       exclude certain long-running sections of
+                                       the documentation (~66% faster)
+      * READTHEDOCS_VERSION        => (used inside docs/conflib/*.py) masquerade as a
```

Comment:
* Only works when READTHEDOCS='True'.  So, if you want to use that variable, need to mention READTHEDOCS as well.
* IMHO undocumented SIMP_VERSION is more useful in practice.
* Probably should document SIMP_CORE_PATH and SIMP_BRANCH here for completeness. (Cut and paste from README)